examples/common/avb.c

```diff
@@ -72,10 +72,17 @@ int pci_connect(device_t * igb_dev)
 
 		printf("attaching to %s\n", devpath);
 		err = igb_attach(devpath, igb_dev);
-		if (err) {
+		if (err || igb_attach_tx(igb_dev)) {
```

Comment:
Oops! I realized I got my rebase wrong last time. Uploaded a new patch, hopefully it's all fine this time. Sorry!